product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx

```diff
@@ -6,9 +6,8 @@ navigation:
 ---
 
 If you plan on deploying a specific version of Postgres Distributed or a specific Postgres distribution or version, you will need to select the appropriate images and image versions.
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
-
-1. Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
-1. [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name). 
-1. [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
+Before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
```

Comment:
I changed this to an unordered list because it doesn't look like order matters.

Reply:
You can't do the last step without having done the preceding two steps.

Reply:
Hi you two. 
Yes, I would like to keep the ordered list here. Especially the last step, assembling your deployment command, can't be done without the first two. Also, although 1 and 2 could be interchanged, I believe from a logical standpoint, it is easier to follow if 1 is done before two.